packages/transformers/posthtml/src/PostHTMLTransformer.js

```diff
@@ -104,15 +104,10 @@ export default (new Transformer({
       plugins: config.plugins,
     });
 
-    if (res.messages) {
-      await Promise.all(
-        res.messages.map(({type, file: filePath}) => {
-          if (type === 'dependency') {
-            return asset.invalidateOnFileChange(filePath);
-          }
-          return Promise.resolve();
-        }),
-      );
+    for (let {type, file: filePath} of res.messages) {
+      if (type === 'dependency') {
+        asset.invalidateOnFileChange(filePath);
+      }
     }
```

Comment:
Let's keep the `if`, not sure if `messages` might be unset sometimes
```suggestion
    if (res.messages) {
      for (let {type, file: filePath} of res.messages) {
        if (type === 'dependency') {
          asset.invalidateOnFileChange(filePath);
        }
      }
    }
```